e2e/scripts/update-mrt-target.js

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const {Command} = require('commander')
+const dotenv = require('dotenv')
```

Comment:
It looks like `dotenv` is not explicitly mentioned in our package.json yet. It's currently available because of other dependencies.

Reply:
I added dotenv to e2e-scripts package.json.

Thanks for catching that